plugins/modules/gitlab_group_access_token.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2024, Zoran Krleza (zoran.krleza@true-north.hr)
+# Based on code:
+# Copyright (c) 2019, Guillaume Martinez (lunik@tiwabbit.fr)
+# Copyright (c) 2018, Marcus Watkins <marwatk@marcuswatkins.net>
+# Copyright (c) 2013, Phillip Gentry <phillip@cx.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+module: gitlab_group_access_token
+short_description: Manages GitLab group access tokens
+description:
+  - Creates, gets and revokes group access tokens.
+author:
+  - Zoran Krleza (@pixslx)
+requirements:
+  - python-gitlab >= 3.1.0
+extends_documentation_fragment:
+  - community.general.auth_basic
+  - community.general.gitlab
+  - community.general.attributes
+notes:
+  - Access tokens can not be changed. If a parameter needs to be changed, acceess token has to be recreated.
```

Comment:
```suggestion
  - Access tokens can not be changed. If a parameter needs to be changed, an access token has to be recreated.
    Whether tokens will be recreated is controlled by the O(recreate) option, which defaults to V(false).
```
The content of the second line depends on how the parameter is changed, see further below.